src/systems/sound-effects-system.js

```diff
@@ -0,0 +1,75 @@
+/* global fetch */
+import TICK from "../assets/sfx/tick.mp3";
+
+function getBuffer(url, context) {
```

Comment:
This can be simplified as
```
function getBuffer(url, context) {
    return fetch(url)
        .then(response => response.arrayBuffer())
        .then(arrayBuffer => context.decodeAudioData(arrayBuffer))
}
```